backend/src/modules/transactions/transactions.service.ts

```diff
@@ -70,20 +70,42 @@ export class TransactionsService {
     return temp;
   }
 
+  async findTransactions(
+    date: string,
+    number: number,
+  ): Promise<Transactions[]> {
+    if (number === undefined) {
+      number = 10;
```

Comment:
Sitam puikiam kodo gabalui, yra skirtas obscure feature vadinamas default function arguments :) You can look it up on the [internet](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters)